**Task #03:** Exception outputs a custom error message